src/home_robot/home_robot/agent/ovmm_agent/ovmm_agent.py

```diff
@@ -63,8 +69,16 @@ def __init__(self, config, device_id: int = 0, obs_spaces=None, action_spaces=No
                 config,
                 config.AGENT.SKILLS.NAV_TO_OBJ,
                 device_id=device_id,
-                obs_spaces=obs_spaces,
-                action_spaces=action_spaces,
+                obs_spaces=None,
```

Comment:
How does it work without the obs and action space?

Reply:
It's hardcoded since right now we haven't figured out how to load them from a config. Earlier they were coming from the habitat config which we won't have access to on Stretch.